Lesson_4/user-portait.js

```diff
@@ -0,0 +1,45 @@
+document.addEventListener('DOMContentLoaded', () => {
+   const userYearOfBirth = Number(prompt('Введіть Ваш рік народження'));
+   const userCity = prompt('В якому місті Ви живете?').toLowerCase();
+   const userFavoriteSport = prompt('Ваш улюблений вид спорту?').toLowerCase();
+
+   const userAge = 2022 - userYearOfBirth;
+
+   const capitalOfCountry = {
+      київ: 'України',
```

Comment:
this is horrible practice to use Cyrillic in keys

Reply:
replaced